sdk/src/Core/Amazon.Runtime/Internal/Auth/AWS4aSigningResult.cs

```diff
@@ -99,6 +120,7 @@ public string RegionSet
         /// <summary>
         /// Returns the full presigned Uri
         /// </summary>
+        [Obsolete("This property is always empty in objects returned by AWS4aSigner. Use the ForQueryParameters property instead, to get the query parameters for a presigned URL.")]
```

Comment:
There is a difference in how the SigV4 and SigV4a signing result types return presigned URLs. I think that obsoleting this SigV4a-only property and unifying with SigV4 is the better choice; for reference, there are [zero usages](https://github.com/search?q=%22.PresignedUri%22+language%3AC%23+NOT+is%3Afork&type=code) of this property outside of the S3 library, which will need the Core bump from the replacement of `AWS4aSignerCRTWrapper` either way.